css-url-image-value-and-simple-length.html

```diff
@@ -0,0 +1,39 @@
+<script>
+
+var url = (function() {
+  var c = document.location.href.split('/');
+  c[c.length - 1] = 'resources/1x1-green.png';
+  return c.join('/');
+})();
+
+var fn = [
+  ["image with CSS OM", function() { element.style.backgroundImage = 'url(resources/1x1-green.png)'; }],
+  ["image with CSS Typed OM", function() { element.styleMap.set("background-image", new CSSURLImageValue(url)); }],
+  ["length with CSS OM", function() { element.style.width = "10px"; }],
+  ["length with CSS Typed OM", function() { element.styleMap.set("width", new CSSSimpleLength(10, 'px')); }]
+];
+
+var element = document.createElement('div');
+document.documentElement.append(element);
+
+var n = 10000;
+
+var sum = [0, 0, 0, 0];
+for (var trial = 0; trial < 100; ++trial) {
+  for (var f = 0; f < 4; ++f) {
```

Comment:
Sorry, I didn't finish typing this comment before...

It's better to have separate benchmark files for different properties, even if you have to duplicate code. This file needs to be focused on testing just one property, and split out the other into a new file. Then you can avoid the array of sums, and have variables for "stringOM" and "typedOM".

I think it's also interesting to record the individual values, so we can compare the variance in time taken (due to garbage collection etc).